Simplified/Network/NYPLCaching.swift

```diff
@@ -183,12 +183,36 @@ extension HTTPURLResponse {
   }
 }
 
+//------------------------------------------------------------------------------
+/// The possible strategies for caching for NYPLNetworkExecutor / Responder.
+///
+/// `ephemeral` caching corresponds to `NYPLSessionConfiguration::ephmeral`.
+/// `default` caching corresponds to the "stock" behavior of URLSession in
+/// processing caching headers, caching responses to memory and disk.
+/// `fallback` caching is like `default`, but the executor will attempt to cache
+/// responses even when these would not be cached by URLSession despite having a
+/// sufficient set of caching headers per https://tools.ietf.org/html/rfc7234 --
+/// see `HTTPURLResponse::modifyingCacheHeaders()`.
+enum NYPLCachingStrategy: NSInteger {
+  case ephemeral
+  case `default`
```

Comment:
Is there a reason why `default` is wrapped in apostrophes?

Reply:
it's because it's a swift keyword, so it's its way to allow that value